pyhive/presto.py

```diff
@@ -79,7 +80,7 @@ class Cursor(common.DBAPICursor):
 
     def __init__(self, host, port='8080', username=None, catalog='hive',
                  schema='default', poll_interval=1, source='pyhive', session_props=None,
-                 protocol='http', password=None):
+                 protocol='http', password=None,auth='NONE',verify=False):
```

Comment:
wouldn't it be best to have as default `verify=True`? deactivating SSL should be explicitly done.